src/iterator.cc

```diff
@@ -205,38 +221,34 @@ void checkEndCallback (Iterator* iterator) {
 
 NAN_METHOD(Iterator::Seek) {
   Iterator* iterator = Nan::ObjectWrap::Unwrap<Iterator>(info.This());
-  iterator->GetIterator();
-  leveldb::Iterator* dbIterator = iterator->dbIterator;
-  Nan::Utf8String key(info[0]);
 
-  dbIterator->Seek(*key);
-  iterator->seeking = true;
+  // TODO: check if slice size > 0?
```

Comment:
There are [comments](https://github.com/vweevers/leveldown/blob/iterator-seek/src/iterator.cc#L383) saying "a Slice can't have length 0". Can someone shine some light on that? What should happen on `seek("")`?